.size-limit.js

```diff
@@ -63,10 +63,9 @@ module.exports = [
     gzip: true,
     limit: '100 KB',
   },
-  // This entry is only here temporarily and will be replaced once we have proper CDN bundles
   {
-    name: '@sentry/replay index.js',
-    path: 'packages/replay/build/npm/esm/index.js',
+    name: '@sentry/replay ES6 CDN Bundle (gzipped + minified)',
```

Comment:
I think we can remove the other two entries for replay then, what do you think? IMHO one is enough for us to track this..? Or at least the non-gzipped one?

Reply:
I'd vote to keep the Webpack entries as they are supposed to show us the size of the NPM package bundled by WebPack whereas this entry tracks the CDN bundle.

Reply:
But yes, we can get probably get rid of the non-gzipped one, agreed.